scripts/template_upload_rhevm.py

```diff
@@ -240,6 +251,34 @@ def templatize_vm(api, template_name, cluster):
         sys.exit(127)
 
 
+# get the domain edomain path on the rhevm
+def get_edomain_path(api, edomain):
+    edomain_id = api.storagedomains.get(edomain).get_id()
+    edomain_conn = api.storagedomains.get(edomain).storageconnections.list()[0]
+    return (edomain_conn.get_path() + '/' + edomain_id,
+            edomain_conn.get_address())
+
+
+def cleanup_empty_dir_on_edomain(path, edomainip, sshname, sshpass):
+    """Cleanup all the empty directories on the edomain/edomain_id/master/vms
+    else api calls will result in 400 Error with ovf not found,
+
+    Args:
+        api: API to chosen RHEVM provider.
+        edomain: Export domain of chosen RHEVM provider.
+        edomainip: edomainip to connect through ssh.
+        sshname: edomain ssh credentials.
+        sshpass: edomain ssh credentials.
+    """
+    print "RHEVM: Deleting the empty directories on edomain/vms file..."
+    ssh_client = make_ssh_client(edomainip, sshname, sshpass)
+    command = 'cd %s/master/vms && find . -type d -empty -delete' % path
```

Comment:
For this and other occurences of this formatting - we prefer to do `''.format()` style of formatting for new code, using the old code only where it may cause problems, like strings containing `{` and `}` on their own ...